docs/deployment/getting-started-locally/getting-started-locally.md

```diff
@@ -0,0 +1,196 @@
+```bash
+> kubectl apply -f <path-to-etcd-cr-yaml>
+```
+
+## 06-Verify the Etcd Cluster
+
+To obtain information on the etcd cluster you can invoke the following command:
+```bash
+> kubectl get etcd -o=wide
+```
+
+We adhere to a naming convention for all resources that are provisioned for an `Etcd` cluster. Refer to [etcd-cluster-components](../../concepts/etcd-cluster-components.md) document to get details of all resources that re provisioned.
+
+### Verify Etcd Pods' Functionality
+
+`etcd-wrapper` uses a [distroless](https://github.com/GoogleContainerTools/distroless) image, which lacks a shell. To interact with etcd, use an [Ephemeral container](https://kubernetes.io/docs/concepts/workloads/pods/ephemeral-containers/) as a debug container. Refer to this [documentation](https://github.com/gardener/etcd-wrapper/blob/9040608291c17799ab33012efce628645c3eeedd/docs/deployment/ops.md#operations--debugging) for building and using an ephemeral container which gets attached to the `etcd-wrapper` pod. 
+
+```bash
+# Put a key-value pair into the etcd 
+> etcdctl put <key1> <value1>
+# Retrieve all key-value pairs from the etcd db
+> etcdctl get --prefix ""
+```
+
+For a multi-node etcd cluster, insert the key-value pair using the `etcd` container of one etcd member and retrieve it from the `etcd` container of another member to verify consensus among the multiple etcd members.
+
+## 07-Updating Etcd CR
+
+`Etcd` CR can be updated with new changes. To ensure that `etcd-druid` reconciles the changes you can opt for one of the below options:
+
+* While setting up druid if you have chosen to switch on `auto-reconcile` via CLI flag `--enable-etcd-spec-auto-reconcile` then any changes to the `Etcd` spec will be automatically reconciled.
+
+* If `--enable-etcd-spec-auto-reconcile` is switched off, then by default `Etcd` spec changes are not reconciled. One can intruct etcd-druid to reconcile an `Etcd` by putting `gardener.cloud/operation: reconcile` annotation on the `Etcd` CR. Once the reconciliation is successfully completed then this annotation will be removed by etcd-druid. You can use the following command to add this annotation:
+  ```bash
+  # Annotate etcd-test CR to reconcile
+  kubectl annotate etcd etcd-test gardener.cloud/operation="reconcile"
+  ```
+
+## 08-Cleaning up the setup
+
+If you wish to only delete the `Etcd` cluster then you can use the following command:
+```bash
+> kubectl delete etcd <etcd-name>
+```
+
+This will add the `deletionTimestamp` to the `Etcd` resource.  At the time the creation of the `Etcd` cluster, etcd-druid will add a finalizer to ensure that it cleans up all `Etcd` cluster resources before the CR is removed. 
+```yaml
+  finalizers:
+  - druid.gardener.cloud/etcd-druid
+```
+
+etcd-druid will automatically pick up the deletion event and attempt clean up `Etcd` cluster resources. It will only remove the finaliser once all resources have been cleaned up.
+
```

Comment:
Please also add a point about `make undeploy` here to uninstall the etcd-druid installation on the cluster.

Reply:
sure. added